tests/test-stripe-api-class.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Class TestStripeAPI
+ *
+ * @package Your_Plugin_Name
+ */
+
+use ThemeIsle\GutenbergBlocks\Plugins\Stripe_API;
+use ThemeIsle\GutenbergBlocks\Tests\StripeHttpClientMock;
+
+// Replace with the correct namespace from your uploaded file.
+
+
+/**
+ * Stripe API test case.
+ */
+class TestStripeAPI extends WP_UnitTestCase {
+
+	/**
+	 * The Stripe_API instance.
+	 *
+	 * @var Stripe_API
+	 */
+	private $stripe_api;
+
+	/**
+	 * Set up the test.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		if ( ! defined( 'OTTER_BLOCKS_VERSION' ) ) {
+			define('OTTER_BLOCKS_VERSION', '1.0.0');
+		}
```

Comment:
Just a NIT, we can move this to the bootstrap class to avoid having to redefine it in some other test in future.